portal-web/test/functional/com/liferay/portalweb/macros/headless/blogs/HeadlessBlog.macro

```diff
@@ -30,4 +31,64 @@ definition {
 		return "${entryId}";
 	}
 
+	macro assertCommentERCWithCorrectValue {
```

Comment:
you can use `BlogPostingAPI._getFieldValueOfExistingBlogPosting()` to get the externalReferenceCode, example:

var response (vía BlogPostingAPI.getBlogPostings(); or passing the response from another query)

```
var existingExternalReferenceCode = BlogPostingAPI._getFieldValueOfExistingBlogPosting(
					fieldName = "externalReferenceCode",
					response = "${response}");
```

Reply:
Great idea !!! Thank u Meg.